Javascript/q41.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>q41</title>
+
+<script>
+  function myFunction() {
```

Comment:
What is `myFunction`? It doesn't tell me anything useful about what this function does.

Reply:
done the changes